app/assets/stylesheets/application.scss

```diff
@@ -136,7 +136,7 @@ a.btn.btn-primary:hover{
 input.btn.btn-primary {
   background-color: #e1a332;
   color: white;
-  width: 60%;
+  width: 10%;
```

Comment:
Hey! You can't change the general button width just to make this look good. You will change every button then. Put another class on it instead ;)